corehq/apps/translations/management/commands/translate_po_files.py

```
@@ -0,0 +1,672 @@
+        """
+        po_objects = self.untranslated_messages if po_objects == [] else po_objects
+        self.translation_obj_map = self._build_translation_obj_map(po_objects)
+        return po_objects
+
+    @cached_property
+    def all_message_objects(self):
+        try:
+            return polib.pofile(self.file_path)
+        except FileNotFoundError:
+            """If the file is not found, we will return an empty list."""
+            raise FileNotFoundError(f"File {self.file_path} not found.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)